src/Medulla/Kubernetes/Operator/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the Medulla Contributors under one or more agreements.
+// The Medulla Contributors licenses this file to you under the Apache 2.0 license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using Controller;
+using KubeOps.Operator;
+using Operator;
+
+namespace DatabaseControllerKubeOps;
+
+public static class Program
+{
+    /// <summary>
+    /// Main
+    /// </summary>
+    /// <param name="args"></param>
+    /// <returns></returns>
+    public static Task<int> Main(string[] args)
+    {
+        Console.WriteLine("Trying to get uri for database service");
+        Console.WriteLine("List Service Uris: ");
+        Console.WriteLine("Service Uri {0} is {1}", "database service", ServiceUris.GetDatabaseServiceUri());
+        Console.WriteLine("Service Uri {0} is {1}", "sync service", ServiceUris.GetDatabaseSyncUri());
```

Comment:
This can fail depending on runtime mode, Console is not always available. Use the [.NET Logger](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-6.0) if needed. Generally avoid logger output in Program.cs

Reply:
Removed